server/telemetry.go

```diff
@@ -19,6 +23,21 @@ const (
 	evCallNotifyAdmin = "call_notify_admin"
 )
 
+var (
+	enterpriseSKUs = []string{model.LicenseShortSkuEnterprise}
+	// currently unused
+	// professionalSKUs = []string{model.LicenseShortSkuProfessional, model.LicenseShortSkuEnterprise}
+
+	// We only need to map events that require a SKU (i.e., licensed features). Anything available on unlicensed
+	// servers will map to null as expected.
+	eventToSkusMap = map[string][]string{
+		"user_start_recording": enterpriseSKUs,
+		"user_stop_recording":  enterpriseSKUs,
+		"live_captions_on":     enterpriseSKUs,
+		"live_captions_off":    enterpriseSKUs,
+	}
```

Comment:
I'm not really sure what the use of this is, if it's a static mapping...?

Reply:
@cpoile What do you mean? Its use is below:

https://github.com/mattermost/mattermost-plugin-calls/blob/e1eff49555a26e00b74ec176e04ebef6b2f4f397/server/telemetry.go#L95-L102

Reply:
I meant, if we know that the `user_start_recording` event is always an enterprise SKU, why do we even need to add the `Skus` field to telemetry. I just thought it was kind of a weird thing to add if it's always the same Sku for those particular events. But I guess it's to make sorting/grouping on the analysis side easier (the alternative would be to sort/group based on a list of event types, which is a lot more cumbersome). So nevermind, sorry for the noise.